### ISSUE TITLE
Thanks for continued work Ahman Woods!

### ISSUE THREAD

**Issue by devanteweary** [LibreVR/Revive#1785]
@ahmanwoods 

Hey bro, didn't know how to contact you but I wanted to say thanks and show appreciation for you working on Revive again.

I know many of us have issues. Personally I can't get PokerStars VR to work anymore.
I don't know if your recent contributions fix my or others' issues, but regardless thank you!

**Comment by ahmanwoods** [comment]
With the latest AppVeyor build, can you boot steamVR, close Revive from the system tray, restart it, then see if your controllers work?

**Comment by ahmanwoods** [comment]
I just tested it and it's working for me. 

The controllers not working without restarting revive sometimes is a known issue, I just haven't had time to look into it much. 

Also, I would direct your thanks to @CrossVR, since he does 99% of things around here 😛

**Comment by devanteweary** [comment]
> I just tested it and it's working for me.
> 
> The controllers not working without restarting revive sometimes is a known issue, I just haven't had time to look into it much.
> 
> Also, I would direct your thanks to @CrossVR, since he does 99% of things around here 😛

Thanks to both of you!

Restarting Revive after SteamVR didn't work for me.
I tried the latest AppVeyor build and I recorded what happened here: [https://youtu.be/endT1vtMcFE](url)

It's about 4 minutes long so here are the main points:

- Tried restarting Revive before and after the game launched.
- Tried restarting revive after SteamVR lauched but before the game.
- Tried closing OVR Advanced Settings (another addon I use).
- Tried launching game from the SteamVR library as well as the Revive dashboard.
- I tried the wiki and copied LibRevive64.dll and openvr_api64.dll into the PokerStars folder. There was no xinput_1_3.dll.
- In the video, you can see the OVR Advanced Settings wrist popup show when I rotate my wrist.
- You can see the loading screen is backwards and sometimes upside down.
- In regards to seeing my controllers, I can't see them but nor can I actually move around.
- You can't tell, but in game it's slightly stuttered when I move my head around. I can't explain it but it kind of hurts my eyes.

I want to say, before you started working on it again, the game wouldn't even load for me. Like, I would load it and it would sit there at "Up Next". So I can tell what you're doing is making progress!

Let me know what else I can do to help!

**Comment by ahmanwoods** [comment]
What headset?

**Comment by devanteweary** [comment]
Oh sorry bet that would help ha!

Quest 2 via VirtualDesktop
(also tried AirLink once)

I saw you just posted a new build so I'll try it when I get home and report back.

**Comment by ahmanwoods** [comment]
If you're on Quest 2, why aren't you just running the title natively?

**Comment by gormanj** [comment]
Yes, thank you so much Ahman Woods for keeping up with this. And of course thank you CrossVR for your continued work. Life would not be the same without Revive.

**Comment by Nekcik** [comment]
Yes! Thank you so much for your continuing to update Revive! I was actually for the first time load into Drop Dead: Duel strike and play it using my Valve Index and it runs nice and smooth although it freezes really bad in multiplayer which if there is someone else in the lobby it sends the other person a message like "player is disconnected" or something. It was unplayable that way but at least the game loaded up finally! I look forward to what else you do with this and hopefully down the road a fix for it.

**Comment by gormanj** [comment]
A friend and I were playing Echo VR with the latest beta and it worked flawlessly.

**Comment by devanteweary** [comment]
> If you're on Quest 2, why aren't you just running the title natively?

Hey, sorry it took a while to get back. 

There are a few reasons I play the PC Oculus version.

- When I started, I didn't know there were different versions so I started on the Quest 2, and I spent some money on props and stuff so now I have money sunk into the account. And it doesn't let you use the same account on Steam.

- The PC version is definitely better than the Quest 2. There's much more detail and straight up missing graphics, effects, etc. in the Quest 2 version. For instance, on the Saloon poker table, they just straight took the "people" out of the bar. 

- Where Revive comes in is that it lets me use Steam apps like OVR Advanced Settings and XSOverlay to do stuff like watch Youtube while playing and chatting with friends, have my Discord window up while playing, move around easier. Just all kinds of stuff. I could play via the Oculus app/store on the PC and get the graphics upgrade, but be missing all the functionality of Steam add ons.

Oh so I wanted to let you know I tried uninstalling everything and not having any addons other than Revive. Then reload Revive after SteamVR loads. I've tried using the Steam Library to load, the Revive Dashboard to load, and even the right-click > Inject... method. It all does the same thing as the video above. Upsidedown and backwards loading screen, then the lobby loads and I can see and hear everything, just can't move or see myself.

Just wanted to keep you updated. I'm on SteamVR v1.23.7.

👍
